examples/aws-sso/Earthfile

```diff
@@ -0,0 +1,85 @@
+VERSION 0.7
+FROM amazon/aws-cli
+ARG --global EARTHLY_CI
```

Comment:
should this default to `false` rather than the empty string?
```suggestion
ARG --global EARTHLY_CI=false
```

Reply:
I've had no problem with this being empty, but I have no problem setting it to false if you think that's better.